crates/biome_js_analyze/src/lint/nursery/use_default_case.rs

```diff
@@ -0,0 +1,195 @@
+use std::ops::Not;
+
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic, RuleSource};
+use biome_console::markup;
+use biome_deserialize_macros::Deserializable;
+use biome_js_syntax::{JsLanguage, JsSwitchStatement};
+use biome_rowan::{syntax::SyntaxTrivia, AstNode};
+use regex::Regex;
+use serde::{Deserialize, Serialize};
+
+declare_rule! {
+    /// Require default cases in switch statements.
+    ///
+    /// See https://eslint.org/docs/latest/rules/default-case
```

Comment:
We should add our own description.
You can use the ESLint description as a starting point.
```suggestion
```

Reply:
I've updated the description to give a little bit more context, let me know what you think